Users/coala_quickstart.rst

```diff
@@ -0,0 +1,64 @@
+Using coala-quickstart to Generate a coala Configuration File
+=============================================================
+
+This document aims to make people aware of coala-quickstart by providing a
+brief introduction about its features and how to quickly generate
+coala configuration files for your projects.
+
+What is coala-quickstart?
+-------------------------
+
+coala-quickstart is a cli tool that helps users to quickly get started with
+coala by generating a ``.coafile`` tailored for their project. The ``.coafile``
+is generated based on the questions answered by the users about their project.
+
+Features
+--------
+
+coala-quickstart offers the following features:
+
+- Out-of-the-box support for projects using various popular languages such as
+  C/C++, Python, JavaScript and many more with built-in check routines.
+- Automatic detection of languages used in your project.
+- Automatic identification of bears that might be relevant for your project and
+  detection of bear settings based on the languages used.
+- A clean and simple interface with a well defined flow.
+
+Installation
+------------
+
+To install the latest stable version run:
+
+::
+
+    $ pip3 install coala-quickstart
+
```

Comment:
you add the clone, cd and then `pip install .` method for latest development version

Reply:
done